policy/modules/admin/tomoyo-auditd.te

```diff
@@ -0,0 +1,32 @@
+policy_module(tomoyo-auditd, 1.0)
+
+########################################
+#
+# Declarations
+#
+
+type tomoyo_auditd_t;
+type tomoyo_auditd_exec_t;
+init_daemon_domain(tomoyo_auditd_t, tomoyo_auditd_exec_t)
+
+type tomoyo_auditd_etc_t;
+files_config_file(tomoyo_auditd_etc_t)
+
+type tomoyo_auditd_var_log_t;
+logging_log_file(tomoyo_auditd_var_log_t)
+
+########################################
+#
+# Tomoyo-auditd local policy
+#
+
+dev_rw_null(tomoyo_auditd_t)
```

Comment:
this is probably redundant. I suspect all "domain" can rw null

Reply:
Yes, not needed anymore.